gr-digital/grc/digital_ofdm_sync_sc_cfb.block.yml

```diff
@@ -42,4 +43,20 @@ templates:
     callbacks:
       - set_treshold(${threshold})
 
+cpp_templates:
+    includes: ['#include <gnuradio/digital/ofdm_sync_sc_cfb.h>']
+    declarations: 'digital::ofdm_sync_sc_cfb::sptr ${id};'
+    make: |-
+        this->${id} = digital::ofdm_sync_sc_cfb::make(
+          ${fft_len},
+          ${cp_len},
+          ${use_even_carriers},
+          ${threshold});
+    link: ['gnuradio-digital']
+    callbacks:
+      - set_treshold(${threshold})
```

Comment:
Indentation typo, causes error on GRC startup

Reply:
Actually, I checked out the master branch of gnuradio and found that this particular indentation and a typo was already there. So, Fixed that :)